src/clients/QBittorrent.ts

```diff
@@ -224,6 +225,14 @@ export default class QBittorrent implements TorrentClient {
 		autoTMM: boolean;
 		category: string;
 	}> {
+		const { dataDirs } = getRuntimeConfig();
+		if (dataDirs.length > 0) {
+			const save_path: string = join(searchee.path, "/..");
+			const isComplete: boolean = true;
+			const autoTMM: boolean = false;
+			const category: string = " ";
+			return {save_path, isComplete, autoTMM, category}
```

Comment:
nit: can we inline these variables into the return object? also, it might be clearest to just omit the category property entirely in this case (and make it optional in the type) since it doesn't get used.